src/main/java/gregtech/GregTechMod.java

```diff
@@ -44,11 +54,12 @@ public GregTechMod() {
     public void onConstruction(FMLConstructionEvent event) {
         moduleManager = ModuleManager.getInstance();
         GregTechAPI.moduleManager = moduleManager;
-        OreGlob.setCompiler(input -> new OreGlobParser(input).compile());
         moduleManager.registerContainer(new GregTechModules());
         MinecraftForge.EVENT_BUS.post(new ModuleContainerRegistryEvent());
         moduleManager.setup(event.getASMHarvestedData(), Loader.instance().getConfigDir());
         moduleManager.onConstruction(event);
+
+        OreGlob.setCompiler((expr, ignoreCase) -> new OreGlobParser(expr, ignoreCase).compile());
```

Comment:
I know this was here before but could this get moved to CoreModule#onConstruction instead?